src/plugins/mer/mersettings.cpp

```diff
@@ -198,6 +204,20 @@ void MerSettings::setAskBeforeClosingVmEnabled(bool enabled)
     emit s_instance->askBeforeClosingVmEnabledChanged(s_instance->m_askBeforeClosingVmEnabled);
 }
 
+FileName MerSettings::deviceModelFileName()
```

Comment:
deviceModel**s**FileName.And shouldn't it contain the word "global"? We will have the user-specific config file as well, right?

Reply:
Added:
`static Utils::FileName globalDeviceModelsFileName();`
`static Utils::FileName deviceModelsFileName();`
to MerSettings.